### PR TITLE
BAT Points Round 2

### PR DESCRIPTION
**Pull request by ryanml**
Fixes brave/brave-browser#6654

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [x] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Test Plan:
1. Enable rewards
2. Wait for initilization
3. Close browser
4. Open Preferences file and search for countryid_at_install
5. Replace current value with 19024
6. Restart Browser
Specification laid out in issue

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: components/brave_rewards/resources/extension/brave_rewards/_locales/en_US/messages.json [hunk 1]
```diff
@@ -15,9 +15,9 @@
     "message": "Get paid for viewing ads and pay it forward to support your favorite content creators.",
     "description": "Entry text for welcome screen in the panel, existing user"
   },
-  "welcomeDescTwo": {
```
Comment (NejcZdovc) [inline components/brave_rewards/resources/extension/brave_rewards/_locales/en_US/messages.json]: why was this translation removed?

File: components/brave_rewards/resources/page/components/adsBox.tsx [hunk 2]
```diff
@@ -283,13 +285,14 @@ class AdsBox extends React.Component<Props, State> {
     const historyEntries = adsHistory || []
     const rows = this.getAdHistoryData(historyEntries, savedOnly)
     const notEmpty = rows && rows.length !== 0
+    const tokenString = onlyAnonWallet ? 'points' : 'tokens'
```
Comment (NejcZdovc) [inline components/brave_rewards/resources/page/components/adsBox.tsx]: why is this one not translated?

File: components/brave_rewards/resources/ui/components/panelWelcome/index.tsx [hunk 3]
```diff
@@ -75,6 +77,10 @@ export default class PanelWelcome extends React.PureComponent<Props, {}> {
       }
     }
 
+    const descriptionLocale = onlyAnonWallet && this.locale.desc === 'welcomeDescTwo'
```
Comment (NejcZdovc) [inline components/brave_rewards/resources/ui/components/panelWelcome/index.tsx]: maybe we should do this in `locale` function?

File: components/brave_rewards/resources/ui/components/walletSummary/index.tsx [hunk 4]
```diff
@@ -91,6 +91,9 @@ export default class WalletSummary extends React.PureComponent<Props, {}> {
     let current = 0
 
     list.forEach((item, index) => {
+      const translatedLabel = item.translation === 'tokenGrantClaimed' && onlyAnonWallet
```
Comment (NejcZdovc) [inline components/brave_rewards/resources/ui/components/walletSummary/index.tsx]: can we do this in `generateList`?

Comment (ryanml) [reply components/brave_rewards/resources/ui/components/walletSummary/index.tsx]: done

File: components/brave_rewards/resources/ui/components/panelWelcome/index.tsx (hunk 3 above)
Comment (ryanml) [reply components/brave_rewards/resources/ui/components/panelWelcome/index.tsx]: moved

File: components/brave_rewards/resources/page/components/adsBox.tsx (hunk 2 above)
Comment (ryanml) [reply components/brave_rewards/resources/page/components/adsBox.tsx]: fixed

File: components/brave_rewards/resources/extension/brave_rewards/_locales/en_US/messages.json (hunk 1 above)
Comment (ryanml) [reply components/brave_rewards/resources/extension/brave_rewards/_locales/en_US/messages.json]: oversight, added back in

**Pull request merged**